src/custom/theme/cowSwapTheme.tsx

```diff
@@ -111,9 +115,9 @@ function themeVariables(darkMode: boolean, colorsTheme: Colors) {
       `,
     },
     appBody: {
-      boxShadow: `4px 4px 0px ${colorsTheme.black}`,
-      borderRadius: '16px',
-      border: `3px solid ${colorsTheme.black}`,
+      boxShadow: `none`,
+      borderRadius: '1rem',
```

Comment:
I suspect @biocom will say: "use `px` instead of `rem` for consistency with the rest of the app"

Reply:
Yes.